components/text/src/vwc-text.scss

```diff
@@ -1,49 +1,80 @@
+
+@use 'sass:list';
 @use '@vonage/vvd-typography/scss/typography' as typography;
 @use '@vonage/vvd-design-tokens/build/scss/typography-variables/web' as typography-scale;
 @use '@vonage/vvd-foundation/scss/mixins/connotation/config' with (
 	$connotations-set: primary cta announcement info success alert,
-	$default: primary,
 );
 @use '@vonage/vvd-foundation/scss/mixins/connotation' as connotation;
 
+$font-faces: ();
+
+@each $key, $value in typography-scale.$typography-category-list {
+	$font-faces: list.join($font-faces, $key, $separator: comma);
+}
+
+@function get-default-selector($font-faces) {
+	$classes: ();
+
+	@each $font-face in $font-faces {
+		$classes: list.join($classes, '.font-face-#{$font-face}', $separator: comma);
+	}
+
+	@return '&:not(#{$classes})';
+}
+
+$default-selector: get-default-selector($font-faces);
+
+%not-tight {
+	&:not(.tight) {
+		display: block;
```

Comment:
@rachelbt what's the criterion for display block of text?

Reply:
as in the user agent stylesheet